tap_stripe/__init__.py

```diff
@@ -1078,6 +1101,25 @@ def sync():
                 if STREAM_TO_TYPE_FILTER.get(stream_name):
                     sync_event_updates(stream_name, Context.is_sub_stream(stream_name))
 
+def get_date_window_size(param, default_value):
+    """
+    Get date_window value from config, if the value is passed.
+    Else return the default value.
+    """
+    window_size = Context.config.get(param)
+
+    # If window_size is not passed in the config then set it to the default(30 days)
+    if window_size is None:
+        return default_value
+
+    if ((type(window_size) in [int, float]) or
```

Comment:
Add code comment

Reply:
Added code comment for mentioned statement.